docs/reference/api/rich-text.md

```diff
@@ -75,8 +76,10 @@ Note that:
 
 * The actual markup is passed in the `html` sub-property.
 
-* Any images found during import are imported into the system as Apostrophe image pieces. This means they will appear in the media library alongside images added to Apostrophe in the usual way.
+* Any images found during import are imported into the system as Apostrophe image pieces and automatically converted to the proper figure format with optional captions. This means they will appear in the media library alongside images added to Apostrophe in the usual way.
 
 * Any `alt` attributes found are also imported.
 
+* The optional `imageTags` sub-property accepts an array of tag piece `_id`s that will be applied to any image(s) imported during the process. These tags will be added to the imported images in the media library.
```

Comment:
"accepts an array of tag piece `_id`s" should be more like 
> accepts an array of existing `@apostrophecms/image-tag`    `_id`s

It can be only this piece type and only existing records is my point.